elements/pl-integer-input/pl-integer-input.py

```diff
@@ -131,6 +134,9 @@ def render(element_html, data):
 
             html_params['suffix'] = suffix
             html_params['a_sub'] = numpy.base_repr(a_sub, base) if base > 0 else data['raw_submitted_answers'].get(name, str(a_sub))
+            if data['raw_submitted_answers'].get(name, 'X').strip() == '':
+                html_params['a_sub'] = '(blank)'
+
```

Comment:
If I'm understanding correctly, this behaves differently than pl-matrix-input? That is, pl-matrix-input will convert a blank to 0 (by default) and display this 0 value, so the student sees it. But pl-integer-input will treat the blank value as 0 (by default) but display `(blank)`. Would it be better to follow the pl-matrix-input convention, both for consistency and so the student actually sees the value that's being used? Especially if the `blank-value` is not zero, I think it might be helpful to show it.

Reply:
Done. I also changed the example to make this clearer.